lib/include/event-driven/vDrawSkin.h

```diff
@@ -0,0 +1,1159 @@
+
+                        int id=sensorConfig.get(1).asInt();
+                        double x=sensorConfig.get(2).asDouble();
+                        double y=sensorConfig.get(3).asDouble();
+                        double orientation=sensorConfig.get(4).asDouble();
+                        double gain=sensorConfig.get(5).asDouble();
+                        int    mirror=sensorConfig.get(6).asInt();
+                        int    layoutNum=sensorConfig.get(7).asInt();
+
+                        handL.get_data(id,x,y,orientation,gain,mirror,layoutNum);
+                    }
+                }
+                if(check==2){
+                    yInfo("Building taxel map from file");
+                    for (auto it =handL.repr2TaxelList.begin();it!=handL.repr2TaxelList.end();it++){
+                        for (auto it2 =  it->second.begin(); it2 !=it->second.end(); it2++){
+                            pos.insert(std::pair<int,std::tuple<double, double>>(*it2,handL.makeMap(*it2)));
+                        }
+                    }
+                    xoffset=handL.xoffset;
+                    yoffset=handL.yoffset;
+                    scaling =handL.scaling;
+                    radius = handL.radius;
+                    noise = handL.noise;
+                    max_value = handL.max_value;
+                }
+                else {
+                    yWarning("Unable to build the taxel map from file");
+                    pos = default_pos;
+                }
+            }
+
+        };
+
+};
+
+class isoDrawSkin : public isoDraw {
+
+    private:
+
+    loadMap tmap;
+
+    int radius;
+
+    public:
+
+    void initialise(){
+
+
+        Xlimit = 800;
+        Ylimit = 800;
+
+        isoDraw :: initialise();
+        tmap.initialise("leftHand");
+
+        radius = tmap.radius-1;
+
+    };
+
+    static const std::string drawtype;
+    virtual void draw(cv::Mat &image, const ev::vQueue &eSet, int vTime);
+    virtual std::string getDrawType();
+    virtual std::string getEventType();
+};
+
+
+
+
+class skinDraw : public vDraw {
+
+private:
+
+    loadMap tmap;
+    cv::Mat baseimage;
+
+    int radius;
+
+public:
+
+    void initialise(){
+
+        Xlimit = 800;
+        Ylimit = 800;
+
+        baseimage = cv::Mat(cv::Size(Xlimit, Ylimit), CV_8UC3);
+        baseimage.setTo(255);
+        tmap.initialise("leftHand");
+        radius = tmap.radius;
+
+        int x;
+        int y;
+
+        yInfo("scaling = %d", tmap.scaling );
+        yInfo("xoffset = %d", tmap.xoffset );
+        yInfo("yoffset =%d", tmap.yoffset );
+
+        for (auto it = tmap.pos.begin(); it != tmap.pos.end(); it++)
+        {
+            x = tmap.xoffset + tmap.scaling* std :: get<0>(it->second);
+            y =  tmap.yoffset +tmap.scaling* std :: get<1>(it->second);
+            cv::Point centr_all(x, y);
+            cv::circle(baseimage, centr_all, radius, black,1, CV_AA);
```

Comment:
This is the only part that really changes from the previous implementations of this class